superset/models/core.py

```diff
@@ -187,6 +187,10 @@ def allows_cost_estimate(self) -> bool:
             and cost_estimate_enabled
         )
 
+    @property
+    def explore_database_visualization_id(self) -> int:
```

Comment:
lets keep it simple, just do explore_database_id

also change code https://github.com/apache/incubator-superset/blob/c7618ee54b62760ac5f738996379de0dcd5e326e/superset/views/database/views.py#L190 here to use this property